src/utilities/_visually-hidden.scss

```diff
@@ -3,7 +3,13 @@
     @include govuk-visually-hidden;
   }
 
+  /// @deprecated Deprecated as of release 1.1, replaced by `.govuk-visually-hidden-focusable`
   .govuk-visually-hidden-focussable {
     @include govuk-visually-hidden-focusable;
   }
+
```

Comment:
Can we do

.className,
.secondClassName {
  @include ...
}

Or does that run into problems?

Reply:
Tried, but Sass docs struggles with that when @ ing the deprecation

Reply:
Also I think by generally defining the deprecated code separately make it easier to remove on major releases. We're doing the same with the font baseline adjustments.

Reply:
Sounds good